### ISSUE TITLE
tests: Policy Enforcement  should be set to 'Disabled' since policy enforcement was set to never

### ISSUE THREAD

**Issue by tgraf** [cilium/cilium#1399]
```
04:56:58 ------------------------------------------------------------------------
04:56:58                           K8s Test Failed
04:56:58 Policy Enforcement  should be set to 'Disabled' since policy enforcement was set to never be enabled
04:56:58 
04:56:58 ------------------------------------------------------------------------
```

**Comment by ianvernon** [comment]
What build was this from @tgraf? Was it from before `0d09b461d08417646e4d50c9a85f021f30288d47` was merged? I saw this issue as part of PR #1385 and found that `0d09b461d08417646e4d50c9a85f021f30288d47` fixed it.

**Comment by tgraf** [comment]
This is no longer appearing in master. Looks like this report was from a build pror to #1385. Closing for now.
